app/test/frontend/static_files_test.dart

```diff
@@ -208,18 +208,18 @@ void main() {
     test('script.dart.js and parts size check', () {
       final file = cache.getFile('/static/js/script.dart.js');
       expect(file, isNotNull);
-      expect((file!.bytes.length / 1024).round(), closeTo(324, 1));
+      expect((file!.bytes.length / 1024).round(), closeTo(343, 10));
```

Comment:
1kB sensitivity seems to be more useful to track size changes. With 10kB we could have a change that pushes it up with 9.5kB and then months later a small one pushes it up with 1kB, and we would be bummed.

In this case the small threshold is also a good reminder that this component could be loaded asynchronously, and we should try to skip the extra increase on the main JS file.

Let's add a TODO somewhere.

Reply:
fine I'll set it at 2kb..

I kind of dislike this test, because it's annoying to find it failing so late in the process.

That said, I see the point.

-----------------

> In this case the small threshold is also a good reminder that this component could be loaded asynchronously, and we should try to skip the extra increase on the main JS file.

Yeah, I could see a point in every widget being a thing is asynchronously loaded :rofl: